### PR TITLE
feat: add table copied file key expire time

### PR DESCRIPTION
**Pull request by lichuang**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

feat: add table copied file key expire time

Fixes https://github.com/datafuselabs/databend/issues/8514

### CODE REVIEW

**Review by drmingdrmer — APPROVED** [review]

File: src/query/service/src/interpreters/interpreter_copy_v2.rs [hunk 1]
```diff
@@ -42,7 +42,7 @@ use crate::sql::plans::Plan;
 use crate::storages::stage::StageTable;
 
 const MAX_QUERY_COPIED_FILES_NUM: usize = 50;
-
+const TABLE_COPIED_FILE_KEY_EXPIRE_AT: Option<u64> = Some(7);
```
Comment (Xuanwo) [inline src/query/service/src/interpreters/interpreter_copy_v2.rs:45]: Is there a list of all the magic numbers we are using? How do we prevent the use of duplicated numbers here?

File: src/query/service/src/interpreters/interpreter_copy_v2.rs [hunk 2]
```diff
@@ -87,7 +87,7 @@ impl CopyInterpreterV2 {
         let req = UpsertTableCopiedFileReq {
             table_id,
             file_info: copy_stage_files.clone(),
-            expire_at: None,
+            expire_at: TABLE_COPIED_FILE_KEY_EXPIRE_AT,
```
Comment (drmingdrmer) [inline src/query/service/src/interpreters/interpreter_copy_v2.rs:90]: `expire_at` is the unix timestamp since 1970 in seconds.

**Review by drmingdrmer — CHANGES_REQUESTED** [review]

**Comment by drmingdrmer** [comment]
https://github.com/datafuselabs/databend/blob/06466d607157637035b8848eb52641bbf8737d7e/src/meta/types/src/seq_value.rs#L29

**Comment by BohuTANG** [comment]
Is this PR ready to merge?

File: src/query/service/src/interpreters/interpreter_copy_v2.rs (hunk 1 above)
Comment (lichuang) [reply src/query/service/src/interpreters/interpreter_copy_v2.rs]: This magic number limit in the file only, does not exported outside as public.

**Comment by BohuTANG** [comment]
@mergify update

**Comment by Xuanwo** [comment]
@Mergifyio rebase

**Comment by Xuanwo** [comment]
Strange. Why @PsiACE showed up here...

File: src/query/service/src/interpreters/interpreter_copy_v2.rs (hunk 1 above)
Comment (Xuanwo) [reply src/query/service/src/interpreters/interpreter_copy_v2.rs]: > This magic number limit in the file only, does not exported outside as public.

This is why it's easier to be broken. We could declare the duplicated numbers.

**Comment by PsiACE** [comment]
> Strange. Why @PsiACE showed up here...

I'm not sure what happened, I didn't even perform any action.

**Comment by Xuanwo** [comment]
@mergify update

File: src/query/service/src/interpreters/interpreter_copy_v2.rs (hunk 1 above)
Comment (lichuang) [reply src/query/service/src/interpreters/interpreter_copy_v2.rs]: For now, this magic number is used in this file only, and there is no file to save these const vars.

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**